README.md

```diff
@@ -8,9 +8,9 @@ The Serenata de Amor Operation arose from a combination of needs, from many peop
 
 We are building an intelligence capable of analyzing public spending and saying, with reliability, the possibility of each receipt being unlawful. This information will be used beyond the code, in the world outside of GitHub. Everything is open source from the beginning, allowing others to fork the project when their ideas diverge from the Operation Serenata de Amor.
 
-Our current milestone is to create the means for this kind of automation with the Quota for Exercising Parliamentary Activity (CEAP), from the Brazilian Chamber of Deputies. This job includes the development of APIs, data cleaning and analyses, conception and validation of scientific hyphotheses, confirmation of illicit acts via investigation and reports - to the population and to legal authorities.
+Our current milestone is to create the means for this kind of automation with the Quota for Exercising Parliamentary Activity (CEAP), from the Brazilian Chamber of Deputies. This job includes the development of APIs, data cleaning and analyses, conception and validation of scientific hypotheses, confirmation of illicit acts via investigation and reports - to the population and to legal authorities.
 
-To achieve this goal, unprecedented, we invite everyone to train the intelligence, collect information, cross databases, validate hyphotheses and apply Machine Learning with models competing against each other and getting combined in ensembles with higher precision than any previous option.
+To achieve this goal, unprecedented, we invite everyone to train the intelligence, collect information, cross databases, validate hypotheses and apply Machine Learning with models competing against each other and getting combined in ensembles with higher precision than any previous option.
```

Comment:
Fix typo of a typo fix: hypothes**i**s (not hypothes**e**s) hahaha… (and the same for line 11).